<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowSchoolRadio</class>
 <widget class="QMainWindow" name="MainWindowSchoolRadio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Школьное радио</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameTranslattion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>621</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QCheckBox" name="checkBoxMicrophone">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>251</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Включить микрофон</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTranslation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>581</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Включить трансляцию</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameRecording">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>621</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QPushButton" name="pushButtonRecording">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>581</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Включить запись</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Уровень записи:</string>
     </property>
    </widget>
    <widget class="QLabel" name="TextLabelRecordingLevel">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>140</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
    </widget>
    <widget class="QLabel" name="TextLabelRecording">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Время записи:</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderRecording">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>142</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="TextLabelRecordingDuration">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>123</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameListen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>621</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QPushButton" name="pushButtonPlay">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Прослушать
запись</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPlayList">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>341</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Прослушать
список</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEditDir">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>20</y>
      <width>481</width>
      <height>40</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelAudio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Папка:</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonDir">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>41</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
